bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/SunSpecConstants.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.sunspec.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.openhab.binding.modbus.ModbusBindingConstants;
+
+/**
+ * The {@link SunSpecConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Nagy Attila Gábor - Initial contribution
+ */
+@NonNullByDefault
+public class SunSpecConstants {
+
+    private static final String BINDING_ID = ModbusBindingConstants.BINDING_ID;
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_INVERTER_SINGLE_PHASE = new ThingTypeUID(BINDING_ID,
+            "inverter-single-phase");
+
+    // Block types
+    public static final int COMMON_BLOCK = 1;
+    public static final int INVERTER_SINGLE_PHASE = 101;
+
+    /**
+     * Map of the supported thing type uids, with their block type id
+     */
+    public static final Map<Integer, ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<Integer, ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singletonMap(INVERTER_SINGLE_PHASE, THING_TYPE_INVERTER_SINGLE_PHASE);
```

And let's hope the next entyr comes with OH3 and we can use `Map.of`

Reply:
I've replaced this.
Yes I have implementations ready for 2 other inverter types and 4 meter types ready, but trying to get the basic PR through first.